templates/CRM/UF/Form/AdvanceSetting.tpl

```diff
@@ -102,3 +107,26 @@
     </div><!-- / .crm-block -->
   </div><!-- /.crm-accordion-body -->
 </div><!-- /.crm-accordion-wrapper -->
+
+<script type="text/javascript">
+    {literal}
+    var updateDupeInputs = document.getElementsByName("is_update_dupe");
+    var duplicateURLInput = document.getElementById("for_duplicate_URL_block");
+
+    updateDupeInputs.forEach(function (checkbox, index) {
+        checkbox.addEventListener('change', confirmCheck);
+        if(index == 1)
+            checkbox.dispatchEvent(new Event('change'));
+    });
+
+    function confirmCheck() {
+        if (this.checked && this.value == 1) {
+            console.log("show input");
+            duplicateURLInput.style.display = '';
+        } else if(window.getComputedStyle(duplicateURLInput).display !== 'none') {
+            console.log("hide input");
+            duplicateURLInput.style.display = 'none';
+        }
+    }
+    {/literal}
```

Comment:
Please use a jQuery closure (see https://docs.civicrm.org/dev/en/latest/standards/javascript/#enclosing-your-code ) and remove debug statements.